README.md

```diff
@@ -16,54 +16,64 @@ Poetry replaces `setup.py`, `requirements.txt`, `setup.cfg`, `MANIFEST.in` and `
 based project format.
 
 ```toml
-[tool.poetry]
+[project]
 name = "my-package"
 version = "0.1.0"
 description = "The description of the package"
 
-license = "MIT"
+license = { text = "MIT" }
+readme = "README.md"
+
+# No python upper bound for package metadata
+requires-python = ">=3.9"
 
 authors = [
-    "Sébastien Eustace <sebastien@eustace.io>"
+    { name = "Sébastien Eustace", email = "sebastien@eustace.io" },
 ]
 
+# Keywords (translated to tags on the package index)
+keywords = ["packaging", "poetry"]
+
+dependencies = [
+    # equivalent to ^3.8.1 with semver constraints
+    "aiohttp (>=3.8.1,<4.0.0)",
+    # dependency with extras
+    "requests[security] (>=2.28,<3.0)",
+    # version-specific dependency with prereleases allowed (see below)
+    "tomli (>=2.0.1,<3.0.0) ; python_version < \"3.11\"",
```

Comment:
```suggestion
    "tomli (>=2.0.1,<3.0.0) ; python_version < '3.11'",
```

Maybe, because it looks better without escape sequences?

Reply:
Valid point. Since it is not stated that the file is generated (completely) by Poetry, formatting details may change, and there are different workflows, I would opt for better readability for now.